tests/mocha/registry_test.js

```diff
@@ -99,11 +98,15 @@ suite('Registry', function() {
 
     suite('Does not have', function() {
       test('Type', function() {
-        chai.assert.isNull(Blockly.registry.getClass('bad_type', 'test_name'));
+        assertWarnings(() => {
+          chai.assert.isNull(Blockly.registry.getClass('bad_type', 'test_name'));
+        }, /Unable to find/);
```

Comment:
What is /unable to find/ here?

Reply:
The argument `/Unable to find/` is a regular expression literal.
In `assertWarnings` we use this to compare against the warning that is logged to ensure we have the expected warning.

This code will assert that the warning logged contains the string "Unable to find".

Reply:
Ah cool, I was expecting a string here for some reason.